### PR TITLE
Add tests to the datasqrl-examples repo

### PR DESCRIPTION
**Pull request by Zesky665**
I updated the build pipeline to include tests for each of the examples and I also added a manual run option. 

Once the PR is merged, anyone with the ability to run GitHub Actions on the repo will be able to use it to run tests on one of the examples by using the `Run Workflow` button.
image

### CODE REVIEW

**Comment by Zesky665** [comment]
Do you guys think that adding an option for running all of the tests is warranted?

File: .github/workflows/build-config.yml [hunk 1]
```diff
@@ -28,29 +52,81 @@ jobs:
         password: ${{ secrets.DOCKER_PASSWORD }}
 
     - name: Pull SQRL demo Docker image
-      run: docker pull datasqrl/cmd:v0.5.6
+      run: docker pull datasqrl/sqrl-demo:latest
```
Comment (henneberger) [inline .github/workflows/build-config.yml:55]: Lets move back to cmd:v0.5.6. The sqrl-demo will be removed soon.

**Review by henneberger — CHANGES_REQUESTED** [review]

File: .github/workflows/build-config.yml (hunk 1 above)
Comment (Zesky665) [reply .github/workflows/build-config.yml:55]: Good catch, I'm on it.

Comment (Zesky665) [reply .github/workflows/build-config.yml]: Done

**Pull request merged**